settings.d/shellhooks.yml.example

```diff
@@ -1,6 +1,8 @@
 ---
-# always enable HTTPS only
-#:enabled: https
+# Use of HTTPS only is strongly advised. Exposing shellhooks over http is
+# technically possible but allows anyone to run things on your proxy.
```

Comment:
Technically it doesn't allow anyone to run things, just those with a matching forward + reverse DNS that matches a name in trusted hosts. So if you have `foreman.example.com`, then when you connect from `192.0.2.1` then Smart Proxy will perform a reverse DNS lookup on that IP. If it finds `foreman.example.com` then it does a forward DNS lookup. If that finds `192.0.2.1`, it is considered valid:
https://github.com/theforeman/smart-proxy/blob/fdeef1dc6febcfae22c8d3273cb18d6bdeb31a23/lib/sinatra/authorization.rb#L38

This calls `remote_fqdn`:

https://github.com/theforeman/smart-proxy/blob/fdeef1dc6febcfae22c8d3273cb18d6bdeb31a23/lib/proxy/helpers.rb#L81-L108

I would phrase it as
```suggestion
# Use https for production deployments. http and true only make sense in development
```

Note that I explicitly used https in lower case since that's a valid value. I don't think HTTPS is accepted.

Reply:
Ok amended.